simbrain.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0'?>
+<!DOCTYPE platform SYSTEM "http://simgrid.gforge.inria.fr/simgrid/simgrid.dtd">
+<platform version="4.1">
+  <zone id="zone0" routing="Full">
+    <host id="P0" speed="98.095Mf"/>
+    <host id="H0" speed="76.296Mf"/>
```

Comment:
We could use the speed variable as the production/clearance rate. That is, model production/clearance as computing tasks (let's discuss this).